cassandra_migrate/cli.py

```diff
@@ -40,8 +40,18 @@ def main():
                         help='Connection port')
     parser.add_argument('-u', '--user',
                         help='Connection username')
+    parser.add_argument('--user-environment-variable-name',
```

Comment:
What is the advantage of having configurable environment variables for these two values instead of having a default one?

Reply:
Zen of Python line two, my friend. :-)

Reply:
If we were going to make it implicit, I'd probably do it by allowing any command line option to be overridden by setting `$CASSANDRA_MIGRATE_{uppercased_option_name}`. Do you think that would be preferable?

Reply:
I like that idea, but I wouldn't block this PR because of that. I think just implementing the ones that you're gonna need right now would be enough, but it's always up to you :)